src/views-components/details-panel/details-panel.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from 'react';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from '@material-ui/icons/Close';
+import { StyleRulesCallback, WithStyles, withStyles, Theme } from "@material-ui/core/styles";
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import * as classnames from "classnames";
+import Grid from '@material-ui/core/Grid';
+
+function TabContainer(props: any) {
+	return (
+		<Typography component="div" style={{ padding: 8 * 3 }}>
```

Comment:
Style could be provided by using `classes` prop after wrapping component with `withStyles`. Or to avoid wrapping you could create a method inside DetailsPanel, for example: 
```
renderTabContainer => (children: React.ReactElement[]) =>  
    <Typography className={this.props.classes.tabContainer}>{children}</Typography>
```